src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/fixtures/UserFixture.kt

```diff
@@ -30,5 +31,26 @@ class UserFixture {
       lastLoggedIn = lastLoggedIn,
       inactiveReason = inactiveReason,
     )
+
```

Comment:
Thes prob don't need to be in a companion object - they are just convenience methods on UserFixture

Reply:
Discussed & agreed to leave as is for now